updatecli/updatecli.d/update-deps.yaml

```diff
@@ -0,0 +1,168 @@
+    kind: dockerimage
+    sourceid: podPrivilegedPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/pod-privileged
+  isUserGroupPolicyCurrent:
+    name: Is there a new user-group-psp tag
+    kind: dockerimage
+    sourceid: userGroupPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/user-group-psp
+  isHostPathsPolicyCurrent:
+    name: Is there a new hostpaths-psp tag
+    kind: dockerimage
+    sourceid: hostPathsPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/hostpaths-psp
+  isCapabilitiesPolicyCurrent:
+    name: Is there a new capabilities-psp tag
+    kind: dockerimage
+    sourceid: capabilitiesPolicyTag
+    spec:
+      image: ghcr.io/kubewarden/policies/capabilities-psp
+
+targets:
+  updatekubectlTag:
+    name: Update kubectl image tag
+    kind: yaml
+    sourceid: kubectlImageTag
+    spec:
+      file: "charts/kubewarden-controller/values.yaml"
+      key: "$.preDeleteJob.image.tag"
+  updateAllowPrivilegeEscalationPolicyTag:
+    name: Update allow-privilege-escalation-psp tag
+    kind: yaml
+    sourceid: allowPrivilegeEscalationPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.allowPrivilegeEscalationPolicy.module.tag"
+  updateHostNamespacePolicyTag:
+    name: Update host-namespaces-psp tag
+    kind: yaml
+    sourceid: hostNamespacePolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.hostNamespacePolicy.module.tag"
+  updatePodPrivilegedPolicyTag:
+    name: Update pod-privileged tag
+    kind: yaml
+    sourceid: podPrivilegedPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.podPrivilegedPolicy.module.tag"
+  updateUserGroupPolicyTag:
+    name: Update user-group-psp tag
+    kind: yaml
+    sourceid: userGroupPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.userGroupPolicy.module.tag"
+  updateHostPathsPolicyTag:
+    name: Update hostpaths-psp tag
+    kind: yaml
+    sourceid: hostPathsPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.hostPathsPolicy.module.tag"
+  updateCapabilitiesPolicyTag:
+    name: Update capabilities-psp tag
+    kind: yaml
+    sourceid: capabilitiesPolicyTag
+    spec:
+      file: "charts/kubewarden-defaults/values.yaml"
+      key: "$.recommendedPolicies.capabilitiesPolicy.module.tag"
+
+actions:
+  createUpdatePR:
+    title: "deps: Update policies, kubectl image"
+    kind: "github/pullrequest"
+    scmid: "default"
+    spec:
+      automerge: false
+      mergemethod: squash
+      description: |
+        Automatic update of dependencies: policies and kubectl image
+        This PR has been created by automation.
+
+        NOTE: REMEMBER TO SQUASH MERGE
+      draft: false
+      labels:
+        - "kind/chore"
+        - "dependencies"
```

Comment:
```suggestion
        - "area/dependencies"
```

Reply:
there was no `area/dependencies` (just created it), and renovatebot is using `dependencies`, unified both in https://github.com/kubewarden/helm-charts/pull/407/commits/7dca022417dfe46380e07fb9f537d21b992dda6a.